apps/condo/domains/common/components/LayoutContext.tsx

```diff
@@ -29,7 +29,7 @@ export const useLayoutContext = (): ILayoutContext => useContext<ILayoutContext>
 
 export const LayoutContextProvider: React.FC = (props) => {
     const breakpoints = useBreakpoint()
-    const [isCollapsed, setIsCollapsed] = useState(false)
+    const [isCollapsed, setIsCollapsed] = useState(true)
```

Comment:
The main fix is ​​the addition `hasSider` prop :)

It's not clear how to fix it, because during ssr we don't have `localStorage`. 
I thought that when we have a closed sider and it opens, it's visually better than when it's open and closes.